plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -57,13 +65,59 @@
 """
 
 EXAMPLES = r"""
+- name: Add a Site External EPG
+  cisco.mso.mso_schema_site_external_epg:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: Schema 1
+    template: Template 1
+    external_epg: External EPG 1
+    l3out: L3out1
+    state: present
+  delegate_to: localhost
+
+- name: Remove a Site External EPG
+  cisco.mso.mso_schema_site_external_epg:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: Schema 1
+    template: Template 1
+    external_epg: External EPG 1
+    l3out: L3out1
+    state: absent
+  delegate_to: localhost
+
+- name: Query a Site External EPG
+  cisco.mso.mso_schema_site_external_epg:
+    host: mso_host
+    username: admin
+    password: SomeSecretPassword
+    schema: Schema 1
+    template: Template 1
+    external_epg: External EPG 1
+    l3out: L3out1
+    state: query
+  delegate_to: localhost
+
+- name: Query all Site External EPGs
```

Comment:
Does this display the site external epgs when the l3out is not set?

Reply:
Yes